package.json

```diff
@@ -26,6 +27,7 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^28.1.0",
     "jest-environment-jsdom": "^28.1.0",
-    "sass": "^1.51.0"
+    "sass": "^1.51.0",
+    "typescript": "^4.6.4"
```

Comment:
Fresh install of Next.js app requires some peer deps to be installed explicitly.